app/src/validators/basic.ts

```diff
@@ -1,9 +1,9 @@
 import Joi from 'joi';
 
 import { stringRequiredMaxLengthTrim, stringRequiredYesNo } from './common';
-import { YES_NO } from '../components/constants';
+import { YES_NO, YesNo } from '../components/constants';
```

Comment:
We seem to have created the same thing twice at some point prior. Could you please remove `YesNo` and `YesNoUnsure` from the constants file and use the all caps versions?